src/java.base/share/classes/java/lang/classfile/package-info.java

```diff
@@ -256,17 +256,31 @@
  * unresolved labels) affects internal tools and may cause exceptions later in
  * the classfile building process.
  * <p>
- * Basic syntax control can be achieved using symbolic descriptors. For example
- * using {@link java.lang.constant.ClassDesc} assures the right serial form is
- * used in the particular place inside classfile and also assures the class name
- * has been already syntactically validated.
+ * Syntax validation is applied during symbolic descriptors construction.
```

Comment:
I would drop `syntax` - and just use `Validation`

Reply:
Yes, applied, thanks.